libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -288,8 +288,11 @@ struct PerRenderableBoneUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     };
     BoneData bones[CONFIG_MAX_BONE_COUNT];
 };
-static_assert(sizeof(PerRenderableBoneUib) <= 16384,
-        "PerRenderableUibBone exceed max UBO size");
+
+// ES3.0 only guarantees a max UBO size of 16 KiB, but users may wish to exceed the minspec
+// by adjusting CONFIG_MAX_BONE_COUNT.
+static_assert(sizeof(PerRenderableBoneUib) == CONFIG_MAX_BONE_COUNT * 64,
+        "PerRenderableUibBone has unexpected size");
```

Comment:
This assert becomes a bit useless because the structure itself is an array of `CONFIG_MAX_BONE_COUNT`.

The point of this assert was to make sure to not set a `CONFIG_MAX_BONE_COUNT` that would exceed the minspec we support.

So I think the better/correct solution is to introduce a `CONFIG_MINSPEC_UBO_SIZE` constant the developers can adjust. Same for `CONFIG_MAX_MORPH_TARGET_COUNT`